viewer/src/tools/AxisView/AxisViewTool.ts

```diff
@@ -0,0 +1,321 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import * as THREE from 'three';
+import ComboControls from '@cognite/three-combo-controls';
+import TWEEN from '@tweenjs/tween.js';
+import _ from 'lodash';
```

Comment:
please avoid the import of the whole lodash, it increases bundle size. Use `import foo from 'lodash/foo'` instead. We should add an eslint rule for that :)

Reply:
Okey, thanks for the tip. How is this different from importing * from Three?

Reply:
It doesn't really I belive @christjt, https://discourse.threejs.org/t/tree-shaking-three-js/1349/42

Reply:
yes, it doesn't. The only difference is that we for some reason export the whole THREE from Reveal so it doesn't matter if import it as a whole.